tutorials/analysis/parallel/mp_processSelector.C

```diff
@@ -8,6 +8,7 @@
 ///
 /// \authors Anda Chelba, Gerardo Ganis
 
+#ifdef R__DAVIX_SUPPORT
```

Comment:
We usually disable the test from the CMakeListstxt rather than (seemingly) making them empty.